html/formulario.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="AngularJS Training">
+  <meta name="author" content="JAG">
+  <title>AngularJS Training</title>
+  <link rel="stylesheet" href="../css/styles.css">
+</head>
+<body>
+<header>
+  Formulario
+</header>
+<section>
+  <header>
+    <h1>Workshop Registration</h1>
+    <div class="dropdown">
+      <button class="mainmenubtn">More Actions <span>&#65516;</span></button>
+      <div class="dropdown-child">
+        <a >English</a>
+        <a >Español</a>
+      </div>
+    </div>
+  </header>
+  <article>
+    <u>Register now while seats are available!</u>
+    <form>
+      <div class="form-side">
+        <div class="form-item">
+          <label>
+            First name <span>*</span>
+          </label>
+          <input type="text" title="First name" name="first_name">
+        </div>
+        <div class="form-item">
+          <label>
+            Last name <span>*</span>
+          </label>
+          <input type="text" title="Last name" name="last_name">
+        </div>
+        <div class="form-item">
+          <label>
+            Company Institution <span>*</span>
+          </label>
+          <input type="text" title="Company Institution" name="company">
+        </div>
+        <div class="form-item">
+          <label>
+            Addres <span>*</span>
+          </label>
+          <textarea title="Addres" name="address"></textarea>
+        </div>
+        <div class="form-item">
+          <label>
+            City
+          </label>
+          <input type="text" title="City" name="city">
+        </div>
+        <div class="form-item">
+          <label>
+            State /Province /Region
+          </label>
+          <select name="state">
+            <option value="">-Select-</option>
+          </select>
+        </div>
+        <div class="form-item">
+          <label>
+            Country
+          </label>
+          <select name="country">
+            <option value="">-Select-</option>
+          </select>
+        </div>
+        <div class="form-item">
+          <label>
+            Email <span>*</span>
+          </label>
+          <input type="email" title="E-mail" name="email">
+        </div>
+        <div class="form-item">
+          <label>
+            Phone number <span>*</span>
+          </label>
+          <input type="number" title="Phone number" name="phone">
```

Comment:
`type="number"` will remove any 0 from the beginning. https://www.w3schools.com/tags/att_input_type_tel.asp